packages/editor/src/components/post-author/combobox.js

```diff
@@ -44,13 +45,14 @@ export default function PostAuthorCombobox() {
 		<ComboboxControl
 			__nextHasNoMarginBottom
 			__next40pxDefaultSize
-			label={ __( 'Author' ) }
+			label={ __( 'Authorz' ) }
 			options={ authorOptions }
 			value={ authorId }
 			onFilterValueChange={ debounce( handleKeydown, 300 ) }
 			onChange={ handleSelect }
 			allowReset={ false }
 			hideLabelFromVision
+			isLoading={ isLoading }
```

Comment:
My preference would we to focus this PR solely on the `ComboboxControl` component.

We should add a Storybook example [here](https://github.com/WordPress/gutenberg/blob/f45e35da9bf7c633411f2690853a783472e3cfb7/packages/components/src/combobox-control/stories/index.story.tsx) to allow for easy testing of the component in isolation.

In a separate, follow-up PR, we could then tweak the post author combobox to use the new prop

Reply:
The entire point of this PR is to improve the author selector - changing only the combobox won't actually change anything, shouldn't we aim for PRs that actually improve things?

I can look into adding to the storybook, would this just be a variation with the `isLoading` prop?

Reply:
The `@wordpress/component` package is not only used in the Gutenberg repository, but it's also used by third-party developers. Therefore, we usually try to keep changes to the "library" separate from changes to the Gutenberg app (hence why I suggest testing the changes in isolation in Storybook).

Having smaller and more focused PRs is also easier to review, and understand for a future developer stumbling upon it.

Reply:
> Having smaller and more focused PRs is also easier to review, and understand for a future developer stumbling upon it.

I understand the principal, however this entire PR is currently _less than 35 lines modified_ so I think is already easy to review and understand! Given the _testing instructions_ I provided, I feel having the PR do something makes it **easier** to test and understand.

That said, I sense that you feel strongly about this and I will defer to your more recent experience building here. I will split the combobox and author changes into separate PRs.

Reply:
I created a follow up PR with just the Comobox changes here: 

https://github.com/WordPress/gutenberg/pull/68990

This is a follow up PR to introduce the post author changes: 

https://github.com/WordPress/gutenberg/pull/68991


Let's continue the conversation on those PRs.